apps/financial-aid/web-osk/src/routes/application/ContactInfo/contactInfo.tsx

```diff
@@ -14,24 +14,25 @@ import {
   isEmailValid,
 } from '@island.is/financial-aid/shared/lib'
 
-const EmailForm = () => {
+const ContactInfo = () => {
   const router = useRouter()
 
   const { form, updateForm } = useContext(FormContext)
-  const [hasError, setHasError] = useState(false)
+  const [hasEmailError, setHasEmailError] = useState(false)
+  const [hasPhoneNumberError, setHasPhoneNumberError] = useState(false)
```

Comment:
Does this have to be two separate states, can we not just have hasError? ❌

Reply:
Yess I agree, however you can add that to hasError value, so hasError checks if there is an error, and then the input takes in hasError && validation, 
so for example in the email input 
hasError={hasError && !isEmailValid(form?.emailAddress)}
And phone input 
hasError={hasError && form?.phoneNumber.length !== 7}. 

It is just a suggestion, I think it might be more of scalable solution if there will be added another input there, will we add another useState error for that too?  🤓 😇